international_chars/currency.dtsi

```diff
@@ -0,0 +1,52 @@
+/* Currency*/
+ZMK_UNICODE_SINGLE(curr_dollar, N0, N0, N2, N4)  // $ (Dollar)
+ZMK_UNICODE_SINGLE(curr_cent, N0, N0, A, N2)  // ¢ (Cent)
+ZMK_UNICODE_SINGLE(curr_pound, N0, N0, A, N3)  // £ (Pound)
+ZMK_UNICODE_SINGLE(curr_currency, N0, N0, A, N4)  // ¤ (Currency)
+ZMK_UNICODE_SINGLE(curr_yen, N0, N0, A, N5)  // ¥ (Yen)
+ZMK_UNICODE_SINGLE(curr_armenian_dram, N0, N5, N8, F)  // ֏ (Armenian Dram)
+ZMK_UNICODE_SINGLE(curr_afghani_dram, N0, N6, N0, B)  // ؋ (Afghani)
+ZMK_UNICODE_SINGLE(curr_nko_dorome, N0, N7, F, E)  // ߾ (Nko Dorome)
+ZMK_UNICODE_SINGLE(curr_bengali_rupee1, N0, N9, F, N2)  // ৲ (Bengali Rupee Mark)
+ZMK_UNICODE_SINGLE(curr_bengali_rupee2, N0, N9, F, N3)  // ৳ (Bengali Rupee)
+ZMK_UNICODE_SINGLE(curr_bengali_ganda, N0, N9, F, B)  // ৻ (Bengali Ganda Mark)
+ZMK_UNICODE_SINGLE(curr_gujarati_rupee, N0, A, F, N1)  // ૱ (Gujarati Rupee)
+ZMK_UNICODE_SINGLE(curr_tamil_rupee, N0, B, F, N9)  // ௹ (Tamil Rupee)
+ZMK_UNICODE_SINGLE(curr_thai_baht, N0, E, N3, F)  // ฿ (Thai Baht)
+ZMK_UNICODE_SINGLE(curr_khmer_riel, N1, N7, D, B)  // ៛ (Khmer Riel)
+ZMK_UNICODE_SINGLE(curr_colon, N2, N0, A, N1)  // ₡ (colon)
+ZMK_UNICODE_SINGLE(curr_cruzeiro, N2, N0, A, N2)  // ₢ (Cruzeiro)
+ZMK_UNICODE_SINGLE(curr_french_franc, N2, N0, A, N3)  // ₢ (French Franc)
```

Comment:
I think this might be a duplicate, unless there was something else supposed to go here instead?
```suggestion
```